theme/SubsiteRow.scss

```
@@ -0,0 +1,132 @@
+    border: 1px solid transparent;
+    border-radius: 8px;
+    font-weight: 500;
+    transition:
+      background-color 0.15s ease-out,
+      border-color 0.15s ease-out,
+      box-shadow 0.15s ease-out;
+  }
+
+  > .rp-sidebar-item.rp-sidebar-item--active {
+    --brand: var(--rp-c-brand, #ff351a);
+    // The default theme paints a 15% brand-tinted bg + brand text. Keep the
```

Stylelint (17.12.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)